### PR TITLE
LPS-185674 Add Relationship, Position, and Linked Features to Model Builder

### PR DESCRIPTION
**Pull request by igor-franca**

### CODE REVIEW

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/AccountRestrictionContainer.tsx [hunk 1]
```diff
@@ -16,8 +16,8 @@ import {defaultLanguageId} from '../../utils/constants';
 interface AccountRestrictionContainerProps {
 	errors: FormError<ObjectDefinition>;
 	isApproved: boolean;
+	isLinkedObjectDefinition?: boolean;
```
Comment (igor-franca) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/AccountRestrictionContainer.tsx:19]: here I change to `isLinkedObjectDefinition` in order to follow the pattern of `isApproved` and `isRootDescendantNode`

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/DefinitionNode/DefinitionNode.tsx [hunk 2]
```diff
@@ -152,7 +152,7 @@ export function DefinitionNode({
 				/>
 
 				<NodeFooter
-					isLinkedNode={linkedDefinition}
```
Comment (igor-franca) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/DefinitionNode/DefinitionNode.tsx]: In these TSX components, I kept the same pattern mentioned earlier, so we don't have the same properties in two TSX components with different names.

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/AccountRestrictionContainer.tsx [hunk 3]
```diff
@@ -119,7 +119,7 @@ export function AccountRestrictionContainer({
 					!accountRelationshipFields.length ||
 					disableAccountToggle ||
 					isRootDescendantNode ||
-					linkedDefinition
+					isLinkedObjectDefinition
```
Comment (naraandrade) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/AccountRestrictionContainer.tsx:122]: @igor-franca Please, sort it in alphabetical order.

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/AccountRestrictionContainer.tsx [hunk 4]
```diff
@@ -144,7 +144,7 @@ export function AccountRestrictionContainer({
 					!values.accountEntryRestricted ||
 					disableAccountSelect ||
 					isRootDescendantNode ||
-					linkedDefinition
+					isLinkedObjectDefinition
```
Comment (naraandrade) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/AccountRestrictionContainer.tsx:147]: @igor-franca Please, sort it in alphabetical order.

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/ConfigurationContainer.tsx [hunk 5]
```diff
@@ -27,7 +27,9 @@ export function ConfigurationContainer({
 		: values.system;
 
 	const disabled =
-		isReadOnly || !hasUpdateObjectDefinitionPermission || linkedDefinition;
+		isReadOnly ||
```
Comment (naraandrade) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/ConfigurationContainer.tsx:30]: @igor-franca Please, sort it in alphabetical order.

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/ConfigurationContainer.tsx [hunk 6]
```diff
@@ -73,7 +75,7 @@ export function ConfigurationContainer({
 			/>
 
 			<Toggle
-				disabled={isReadOnly || linkedDefinition}
+				disabled={isReadOnly || isLinkedObjectDefinition}
```
Comment (naraandrade) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/ConfigurationContainer.tsx:78]: @igor-franca Please, sort it in alphabetical order.

**Review by naraandrade — CHANGES_REQUESTED** [review]

**Review by naraandrade — APPROVED** [review]
@igor-franca Please, take a look at the SF error.

**Comment by igor-franca** [comment]
> @igor-franca Please, take a look at the SF error.

The SF erro is related to `WikiPage.testcase` and `MFAFilter.java`

These files were not modified in this PR, the error in SF is not related to the modifications.

**Comment by igor-franca** [comment]
resent [here](https://github.com/brianchandotcom/liferay-portal/pull/140376)

**Pull request closed without merging**